src/main/clojure/clara/rules/compiler.clj

```diff
@@ -1677,7 +1775,8 @@
    fact-type-fn
    ancestors-fn
    activation-group-sort-fn
-   activation-group-fn]
+   activation-group-fn
+   expressions]
```

Comment:
Another place I'd consider renaming as mentioned above. `expressions` is probably the most vague.